capa/rules/cache.py

```diff
@@ -107,6 +168,41 @@ def get_ruleset_content(ruleset: capa.rules.RuleSet) -> List[bytes]:
 
 def compute_ruleset_cache_identifier(ruleset: capa.rules.RuleSet) -> CacheIdentifier:
     rule_contents = get_ruleset_content(ruleset)
+
+    if is_dev_environment():
```

Comment:
we should probably try/catch this to not fail running if something goes wrong

Reply:
Is this in regards to `is_dev_environment`?

Reply:
surrounding all of the related checks most likely